docs/nGQL-tutorial.md

```diff
@@ -0,0 +1,259 @@
+# nGQL Query Language
+
+
+nGQL is the query language of Nebula Graph that allows users to store and retrieve
+data from the graph database. Nebula Graph wants to make its queries easy to learn,
+understand, and use for everyone.
+
+## Key attributes of nGQL
+
+Nebula Graph is committed to create a new query language that specifically deals
+with graph data. nGQL has two attributes that are not available together in any
+other query language out there.
+
+- Declarative: nGQL is a declarative query language, which is very different from
+ the imperative alternatives out there. You declare the pattern that you are looking for. You effectively tell nGQL what you want, rather than how to get it.
+
+- Expressive: nGQL's ASCII-art style syntax provides a familiar, readable way to
+ match patterns of nodes and relationships within graph datasets.
+
+## nGQL Syntax
+
+nGQL key words are case-insensitive but we recommend them written in all caps for easy reading. To help you get a quick understanding of nGQL, we have created a simple graph `myspace_test` with 4 vertices and 3 edges.
+
+### Cluster administration
+
+* Add hosts
+
+Add a single host
+
+```
+ADD HOSTS $storage_ip:$storage_port
+```
+
+Add multiple hosts
+
+```
+ADD HOSTS $storage_ip1:$storage_port1,
+$storage_ip2:$storage_port2,...
+```
+
+**Note:**
+Replace the $storage_ip and $storage_port here according to the local_ip and port in nebula-storaged.conf. Separate the hosts by comma.
+
+
+- Show active hosts
```

Comment:
show hosts..   some hosts are not acrive

Reply:
fixed